src/main/java/Poirot.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Scanner;
+
+public class Poirot {
+    private static Task[] list_actions = new Task[100];
+    private static int last_index = 0;
+
+    public static void echo(String msg) {
+        System.out.println("____________________________________________________________\n");
```

Comment:
I noticed that this was repeated many times. Perhaps it would be good to use a utility function to print the horizontal lines instead.